### DISCUSSION TITLE
any thoughts on decentralization for collaboration features?

### DISCUSSION THREAD

**Discussion by ckemmler** [AppFlowy-IO/AppFlowy#777] [General]
Not getting any response in a week, I prefer to delete this message but I am somehow unable to do so. Hence this useless message.

**Comment by heyakyra** [comment]
See: https://github.com/AppFlowy-IO/AppFlowy/discussions/12#discussioncomment-3410477

**Comment by annieappflowy** [comment]
Hi @ckemmler , sorry for the late reply. I missed this one. To your question on the collaboration feature, if you look at https://github.com/AppFlowy-IO/AppFlowy/tree/main/frontend/app_flowy/packages/appflowy_editor/lib/src/operation, you can see we set up the foundations for supporting online collaboration. You can build decentralized collaboration based off our codebase. We're happy to brainstorm on this topic with you!
